src/transformers/models/llava/modeling_llava.py

```diff
@@ -523,8 +523,9 @@ def forward(
 
             # TODO: @raushan retain only the new behavior after v4.47
             else:
-                n_image_tokens = (input_ids == self.config.image_token_index).sum(dim=-1)[0].item()
-                n_image_features = image_features.shape[1]
+                n_image_tokens = (input_ids == self.config.image_token_index).sum().item()
```

Comment:
I think this slows down inference no? `.item()` induces cuda cpu synch! 
Anyways not the point, but thanks for the fix. 
`n_image_features` takes into account padding? (are image features not padded to the batch?

Reply:
yes, if the image is padded it is usually unpadded before we come to this point, e.g in llava-next. Hm, I don't think the slowdown will be drastic especially since we need to check once per input in the pre-fill stage